docs/java/reference/messaging/MessageRepository.mdx

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 0
+---
+
+# MessageRepository
+
+The `MessageRepository` contains method used to create, edit, get and delete message using the Java Wrapper Library
+
+## Constructors
+
+### constructor
+
+- **new MessageRepository(String projectId, String apiToken, String spaceName)**
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `projectId` | `String` | ProjectId gotten from your Signalwire space e.g `a10d8a9f-2166-4e82-56ff-118bc3a4840f` |
+| `apiToken` | `String` | Api token generated from your Signalwire dashboard e.g `PT9e5660c101cd140a1c93a0197640a369cf5f16975a0079c9`|
+| `spaceName` | `String` | Name of your space e.g `testing` |
+
+## Methods
+
+### create
+
+▸**create(**`String to`, `String from`, `String body`**)**:`NewMessageResponse`
```

Comment:
Can we use the same style used in the `signalwire/docs` repo? E.g.

    ▸ **create**(`String to`, `String from`, `String body`): `NewMessageResponse`

(notice the different spaces / bold)